internal/integrationtests/integrationtests.go

```diff
@@ -0,0 +1,120 @@
+package integrationtests
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/stellar/go/keypair"
+	"github.com/stellar/go/support/log"
+	"github.com/stellar/go/txnbuild"
+
+	"github.com/stellar/wallet-backend/internal/services"
+	"github.com/stellar/wallet-backend/pkg/utils"
+	"github.com/stellar/wallet-backend/pkg/wbclient"
+	"github.com/stellar/wallet-backend/pkg/wbclient/types"
+)
+
+type IntegrationTestsOptions struct {
+	BaseFee           int64
+	NetworkPassphrase string
+	RPCService        services.RPCService
+	SourceAccountKP   *keypair.Full
+	WBClient          *wbclient.Client
+}
+
+func (o *IntegrationTestsOptions) Validate() error {
+	if o.BaseFee < int64(txnbuild.MinBaseFee) {
+		return fmt.Errorf("base fee is lower than the minimum network fee")
+	}
+
+	if o.NetworkPassphrase == "" {
+		return fmt.Errorf("network passphrase cannot be empty")
+	}
+
+	if o.RPCService == nil {
+		return fmt.Errorf("rpc client cannot be nil")
+	}
+
+	if o.SourceAccountKP == nil {
+		return fmt.Errorf("source account keypair cannot be nil")
+	}
+
+	if o.WBClient == nil {
+		return fmt.Errorf("wallet backend client cannot be nil")
+	}
+
+	return nil
+}
+
+type IntegrationTests struct {
+	BaseFee           int64
+	NetworkPassphrase string
+	RPCService        services.RPCService
+	SourceAccountKP   *keypair.Full
+	WBClient          *wbclient.Client
+}
+
+func (i *IntegrationTests) Run(ctx context.Context) error {
+	log.Ctx(ctx).Info("🆕 Starting integration tests...")
+
+	// Step 1: call /tss/transactions/build
+	log.Ctx(ctx).Info("🚧 Building transactions locally...")
+	buildTxRequest, err := i.prepareBuildTxRequest()
+	if err != nil {
+		return fmt.Errorf("preparing build tx request: %w", err)
+	}
+	log.Ctx(ctx).Info("⏳ Calling {WalletBackend}.BuildTransactions...")
+	builtTxResponse, err := i.WBClient.BuildTransactions(ctx, buildTxRequest.Transactions...)
+	if err != nil {
+		return fmt.Errorf("calling buildTransactions: %w", err)
+	}
+	log.Ctx(ctx).Infof("✅ builtTxResponse: %+v", builtTxResponse)
+
+	log.Ctx(ctx).Info("🚧 TODO: feeBumpTx")
```

Comment:
Why are we logging the TODO items in the integration test logs? We can just add a "TODO:.." comment here

Reply:
I'm actively working on the integration tests right now and I wanted to convey the clear information that it's not complete yet, in case someone were to start using this command.

Reply:
I'm converting it to comments in https://github.com/stellar/wallet-backend/pull/155/commits/b4506a6a3e8b52261c358615f67ba5e84528aeec.